menu.js

```diff
@@ -0,0 +1,19 @@
+const icon = document.querySelector('#icon');
+icon.addEventListener('click', () => {
+  const container = document.querySelector('.container');
+  const nav = document.querySelector('nav');
+  const span = document.createElement('span');
+  span.innerHTML += '<span>&times;</span>';
+  span.classList.add('close');
+  nav.appendChild(span);
+  if (nav.style.display === 'inline-block') {
+    nav.style.display = 'none';
+  } else {
+    nav.style.display = 'block';
+    nav.classList.add('nav-style');
+    span.addEventListener('click', () => {
+      container.removeChild(nav);
+    });
+    document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => { container.removeChild(nav); }));
+  }
```

Comment:
- [ ] Great job here 👌 but there is a small issue when I click on `menu-icon` the `mobile-menu` will appear just once. that means If I closed the `mobile-menu` and tried to open it again it doesn't work 🤷‍♂️ that happened because you removed the nav from `DOM` using `container.removeChild(nav)`

https://user-images.githubusercontent.com/45231276/179960520-c55e881d-50d7-4fbf-bfad-a8b906f4ed3e.mp4

> To fix this issue I suggest using `nav.style.display = 'none'` instead of `container.removeChild(nav);` 😉
> For example 

```
  nav.appendChild(span);
  if (nav.style.display === 'inline-block') {
    nav.style.display = 'none';
  } else {
    nav.style.display = 'block';
    nav.classList.add('nav-style');
    span.addEventListener('click', () => {
      nav.style.display = 'none';
    });
    document.querySelectorAll('.nav-link').forEach((n) =>
      n.addEventListener('click', () => {
        nav.style.display = 'none';
      })
    );
  }
``